versioning.md

```diff
@@ -30,6 +28,44 @@ Whether a new version is a patch release, a minor release or a major release sho
 following Semantic Versioning. https://semver.org/
 
 
+## Changelog
+
+All projects should keep a changelog. A file called CHANGELOG.md should be in the root of the repository.
+
+It's format should follow https://keepachangelog.com/en/1.0.0/
+
+To start a new repository that has no releases, the initial changelog should be:
+
+```
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+```
+
+
+## Process for making a new release
+
+These steps depend on each other so carry them out in order.
+
+### Commit to master branch
+
+A special commit should be made to the master branch.
+
+This commit should:
+
+  *  change the version number in setup.py
+  *  change the version number and fix any problems with messages for this release in CHANGELOG.md
+
+### Tag a commit
+
+The GitHub web interface should then be used to create a new tag that points to the new commit on the master branch.
```

Comment:
If there's a reason why `git tag` isn't suitable, it'd be good to explain it here

Reply:
It's just easier, I can comment